libr/debug/debug.c

```diff
@@ -5,7 +5,7 @@
 #include <signal.h>
 
 #if __WINDOWS__
-void w32_break_process(void *);
+void w32_break_process_wrapper(void *);
```

Comment:
this function is not following the naming conventions of the module, so the rules say that we should make them static or start with `__` (double lower dash)